socorro/unittest/external/es/test_supersearch.py

```diff
@@ -1569,33 +1550,33 @@ def test_get_with_number_histogram(self):
                 'term': day_before_int,
                 'count': 1,
                 'facets': {
-                    'product': [
-                        {
-                            'term': 'NightTrain',
-                            'count': 1
-                        },
-                    ],
                     'platform': [
                         {
                             'term': 'Linux',
                             'count': 1
                         }
                     ],
+                    'product': [
+                        {
+                            'term': 'NightTrain',
+                            'count': 1
+                        },
+                    ],
```

Comment:
No, that's just because I was stupid when debugging a test failure and thought it was an array and not a dict. :)